lib/broadside/deploy.rb

```diff
@@ -9,12 +9,14 @@ class Deploy
       :command,
       :instance,
       :lines,
+      :rollback,
+      :scale,
       :tag,
       :target
     )
 
-    def initialize(target, options = {})
-      @target   = target
+    def initialize(target_name, options = {})
+      @target   = Broadside.config.target_from_name!(target_name)
```

Comment:
you can do that with `Hash#fetch` and a block - see my other comment: https://github.com/lumoslabs/broadside/pull/57#discussion_r98870113